content/en/data_observability/integrations/business_intelligence/metabase.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Metabase
+description: Connect Metabase to Datadog Data Observability to view end-to-end lineage from warehouse tables to dashboards.
+further_reading:
+  - link: '/data_observability/data_quality'
+    tag: 'Documentation'
+    text: 'Learn about Data Quality monitoring'
+---
+
+## Overview
+
+Datadog's Metabase integration helps data teams make changes to their data platform without breaking Metabase dashboards, and identify unused cards or dashboards. When Datadog connects, it:
+
+- Pulls metadata from your Metabase environment, including cards and dashboards.
+- Automatically generates lineage between warehouse tables and columns and downstream Metabase cards, and between those cards and downstream dashboards.
```

Comment:
With the two "and"s in the phrase "warehouse tables and columns and downstream Metabase cards," I'm not 100% sure if the generated lineage is between all of them, or if maybe "tables and columns" count as a single thing?